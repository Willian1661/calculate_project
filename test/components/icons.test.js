const iconNumber = require("../../src/components/icons/numbers-icon")
const iconAsterisk = require("../../src/components/icons/asterisk-icon")
const iconClearAll = require("../../src/components/icons/clear-all-icon")
const iconDelete = require("../../src/components/icons/delete-icon")
const iconDevide = require("../../src/components/icons/divide-icon")
const iconEquals = require("../../src/components/icons/equals-icon")
const iconMinus = require("../../src/components/icons/minus-icon")
const iconPercent = require("../../src/components/icons/percent-icon")
const iconperiod = require("../../src/components/icons/period-icon")
const iconPlus = require("../../src/components/icons/plus-icon")
const iconPlusMinus = require("../../src/components/icons/plus-minus-icon")

describe("operators/numbers/especial icons", () =>{
    const icons = [
        iconAsterisk(),iconDevide(),iconPlus(),iconPlusMinus(),iconMinus(),
        iconPercent(),iconClearAll(),iconDelete(),iconEquals(),iconperiod(),
        iconNumber("1"),iconNumber("2"),iconNumber("3"),iconNumber("4"),iconNumber("5"),
        iconNumber("6"),iconNumber("7"),iconNumber("8"),iconNumber("9"),iconNumber("0")
    ]
        it("has passed all the icons correctly", () => {
        expect(icons).toMatchSnapshot()
    });
    
})