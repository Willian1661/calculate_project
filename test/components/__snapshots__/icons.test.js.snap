// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`operators/numbers/especial icons has passed all the icons correctly 1`] = `
[
  "<i class='fa-solid fa-asterisk'></i>",
  "<i class='fa-solid fa-divide'></i>",
  "<i class='fa-solid fa-plus'></i>",
  "<i class='fa-solid fa-plus-minus'></i>",
  "<i class='fa-solid fa-minus'></i>",
  "<i class='fa-solid fa-percent'></i>",
  "<i class='fa-solid fa-c'></i>",
  "<i class='fa-solid fa-delete-left'></i>",
  "<i class='fa-solid fa-equals'></i>",
  "<i class='fa-solid fa-period'>.</i>",
  "<i class="fa-solid fa-1"></i>",
  "<i class="fa-solid fa-2"></i>",
  "<i class="fa-solid fa-3"></i>",
  "<i class="fa-solid fa-4"></i>",
  "<i class="fa-solid fa-5"></i>",
  "<i class="fa-solid fa-6"></i>",
  "<i class="fa-solid fa-7"></i>",
  "<i class="fa-solid fa-8"></i>",
  "<i class="fa-solid fa-9"></i>",
  "<i class="fa-solid fa-0"></i>",
]
`;
